test/e2e/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	osconfigv1 "github.com/openshift/api/config/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	capiv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	namespace = "openshift-cluster-api"
+)
+
+func init() {
+	if err := capiv1alpha1.AddToScheme(scheme.Scheme); err != nil {
+		glog.Fatal(err)
+	}
+
+	if err := osconfigv1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
you are adding osconfigv1 to the scheme but I can't see where it's used, see comment above

Reply:
removed